packages/common/rollup-configs.js

```diff
@@ -59,7 +55,6 @@ export const tsLibConfig = (pkg, inputFile, format = 'esm') => ({
   output: {
     file: 'dist/index.js',
     format,
-    banner: getBanner(pkg),
```

Comment:
@vojtechszocs I don't think we should remove the comment from the JS bundle. Can we do both? If someone doesn't know to look for `build-meta.json` it may be confusing.

Reply:
Our consumers should use webpack to process their host apps and/or plugins into deployable assets, i.e. plugin SDK libraries are not meant to be loaded directly in the browser.

While I think that having a dedicated metadata file is better, we can keep the comment block in the `.js` bundle.

Note that we don't do that for the `.d.ts` bundle, maybe we should?